<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${thread}] [${node}/${function}:${line}]: ${message}"/>

    <node pkg="lost_comms_recovery" type="lost_comms_recovery" name="lost_comms_recovery" clear_params="true" output="screen" respawn="true" >
      <param name="ping_fail_count" value="1"/>
      <param name="recovery_behaviour" value="go_home"/>
      <param name="goal_frame_id" value="map"/>
      <!-- <param name="ips_to_monitor" value="192.168.1.2,192.168.1.3"/>  --><!-- defaults to ROS_MASTER's IP -->
      <param name="ips_to_monitor" value="192.168.190.136"/>
      <!-- INPUT TOPICS -->
      <remap from="recovery_pose" to="recovery_pose" />
      <!-- OUTPUT TOPICS -->
      <remap from="cmd_vel" to="cmd_vel" /> <!-- zero motors -->
      <remap from="joy_arm" to="joy_arm" /> <!-- zero joystick -->
    </node>
</launch>
